packages/wast-parser/src/tokenizer.js

```diff
@@ -113,6 +96,30 @@ function tokenize(input: string) {
 
   const tokens = [];
 
+  /**
+   * Creates a pushToken function for a given type
+   */
+  function pushToken(type: string) {
+    return function(v: string | number, opts: Object = {}) {
+      tokens.push(Token(type, v, line, column, opts));
+    };
+  }
```

Comment:
@xtuc What do you think of this? This way `column` and `line` values are automatically inserted via closure.

Reply:
Yup :+1: